lib/push.js

```diff
@@ -61,8 +61,9 @@ module.exports = function(file, options, callback) {
 
 	function getUri(options) {
 		var packageUri = options.host.replace(/\/?$/, '/');
+		var spaceId = (options.hasOwnProperty('spaceId') == true) ? options.spaceId : "Spaces-1";
 
-		packageUri = url.resolve(packageUri, 'api/packages/raw');
+		packageUri = url.resolve(packageUri, ('api/' + spaceId + '/packages/raw'));
```

Comment:
I have a feeling this is going to break compatability for older pre-spaces versions of octopus. If so we may need to detect spaces feature (either through feature sniffing or version check) and create the url accordingly

Reply:
You are absolutely correct, it would break.  @ryanrousseau came up with an easy solution to this which is to only add the space id when it is provided.  I've updated the code with this check.